settings.gradle.kts

```diff
@@ -15,5 +15,6 @@ pluginManagement {
 rootProject.name = "giskard"
 include(
     "backend",
-    "frontend"
+    "frontend",
```

Comment:
you may have an old version of the gradle configs
@Hartorn has removed these parts

Reply:
I added parts of the task back because Dockerfile still depends on `python-client` tasks. Let me check how to remove it gradually.